pkgs/top-level/all-packages.nix

```diff
@@ -5755,7 +5755,7 @@ with pkgs;
 
   joystickwake = callPackage ../tools/games/joystickwake { };
 
-  juce = callPackage ../development/misc/juce {
+  juce = callPackage ../by-name/ju/juce/package.nix {
```

Comment:
As part of [RFC0140](https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md) we no longer require a definition in `all-packages.nix`. We can remove the `juce` definition and declaration here.

To confirm this is working you should be able to run `nix-build juce` without issue.

Reply:
Just a small mistake on my part, will fix asap.

Reply:
Sorry, I didn't see that we were overriding stdenv for darwin. This looks fine